frontend/views/containers/group-settings/InvitationsTable.vue

```diff
@@ -279,8 +287,21 @@ export default ({
         sbp('okTurtles.events/emit', OPEN_MODAL, 'AddMembers')
       }
     },
-    handleSeeOriginal (inviteItem) {
-      console.log(inviteItem, 'TODO - See Original Proposal')
+    async handleSeeOriginal ({ invitee }) {
+      const key = `proposals/${this.ourUsername}/${this.currentGroupId}`
+      const archivedProposals = await sbp('gi.db/archive/load', key) || []
+      const proposalItemExists = archivedProposals.length > 0 || archivedProposals.some(entry => {
+        const { data } = entry[1]
+
+        return data.proposalType === PROPOSAL_INVITE_MEMBER &&
+          data.proposalData.member === invitee
+      })
```

Comment:
Since there is no validation in invitee `Full name`, I think checking existence using `inviteSecret` is more reasonable.
You can just add to check `inviteSecret` in line #298.

Reply:
@Silver-IT I'm not entirely sure what you meant by 'Since there is no validation in invitee Full name' but these lines check if the archived proposals of the app has the `PROPOSAL_INVITE_MEMBER` item for a particular user.
`inviteSecret` field exists for `anyone` invite (as far as it seems) so I reckon checking the existence of that value can't be useful in this case.

Reply:
What I mean is that we can make two proposals to add two member `Alex Jin`. So I don't think the `invitee full name` shouldn't be used as one of the identifiers of the proposal.

Reply:
@Silver-IT Oh, now it's clear what you meant. yeah that's right and I think in that kind of case, additional check of whether the creator of the proposal is the current user would achieve our need here. what's your thought?

Reply:
One creator can add two members with same name, can't him?

Reply:
@Silver-IT That's technically possible but why would anyone do that..? why does there has to be two different active invites for the same person?

Reply:
@taoeffect, what do you think?

Reply:
@Silver-IT Btw, agreed  using `inviteSecret` is another good way to identify a proposal so made a change in the following commit here :)
https://github.com/okTurtles/group-income/pull/1649/commits/f8639e00a5a67e1ddff4813559d975c4608ae09a

Reply:
Thank you, @SebinSong.